kolibri/utils/cli.py

```diff
@@ -502,6 +503,26 @@ def start(port, background):
     Start the server on given port.
     """
 
+    start_with_ready_cb(port, background, ready_cb=_on_ready)
+
+
+def _on_ready(urls):
+    if not urls:
+        logger.error(
+            "Could not detect an IP address that Kolibri binds to, but try "
+            "opening up the following addresses:\n"
+        )
+        urls = [
+            "http://{}:{}".format(ip, bind_port) for ip in ("localhost", "127.0.0.1")
```

Comment:
`bind_port` is not defined. Maybe the partial was imported to do that and you forgot it?

Something like:
```
start_with_ready_cb(port, background, ready_cb=partial(_on_ready, port))

def _on_ready(bind_port, urls):
```

Reply:
Thanks for spotting this! Indeed, I ended up breaking some things when I reorganized this yesterday :) I changed server.py to pass `bind_port` and `bind_addr` to the `on_ready` callback, and that should take care of it.